widgetastic_manageiq/__init__.py

```diff
@@ -1356,11 +1356,12 @@ def find_row_on_pages(self, table, *args, **kwargs):
                                          .format(kwargs, table))
 
 
-def PaginationPane():  # noqa
+def PaginationPane(*args, **kwargs):  # noqa
```

Comment:
oh your right. forgive me my forgetfulness. I've just added Pagination pane to web_ui.
So, it really makes sense.